backend/src/main/java/com/soen341/instagram/service/impl/CommentService.java

```diff
@@ -158,4 +128,38 @@ private Account getCurrentUser()
 		}
 	}
 
+	// like service
+
+	public int likeComment(final long commentId)
+	{
+		final Comment comment = findComment(commentId);
+		final Set<Account> likedBy = comment.getLikedBy();
+		final boolean addedSuccessfully = likedBy.add(getCurrentUser());
```

Comment:
I know the `getCurrentUser` method in this class is not your code but could you remove it and instead use the one in `backend/src/main/java/com/soen341/instagram/utils/UserAccessor.java` (refer to my previous comment)